### DISCUSSION TITLE
The "--min-battery" option in version 35-36 does not work

### DISCUSSION THREAD

**Discussion by liubkkkko** [grahampugh/erase-install#517] [General]
Greetings, community. I try to use the --min-battery option in 35-36 version and it doesn't work. I also do not reject the fact that I do not specify this parameter correctly, so I am attaching a screenshot of how it is configured for me on jamf. I use --min-battery 70, on the test macbook 55% and not connected to the charger. As a result, I get the script working without the --check power option at all
![008](https://github.com/grahampugh/erase-install/assets/104902816/cd6a971f-a1f6-4573-b2e7-27a5e38e3d04)

**Comment by grahampugh** [comment]
Please see the note in the wiki about using Jamf parameters (assuming you have added erase-install.sh directly from Jamf, which is not clear from the screenshot). https://github.com/grahampugh/erase-install/wiki/6.-Use-in-Jamf-Pro#option-2-upload-erase-installsh-into-jamf-pro-and-use-policy-script-parameters 

In short, you need to use an equals sign for parameters with a value if you're adding the script directly to Jamf.

Therefore you need to change to: `--rebootdelay=300`, `--language=ua` and `--min-battery=70`.

Additionally, you need to change `--check power` to `--check-power`.

**Reply by liubkkkko** [reply]
Graham, thank you.
Our problems always arise because of ourselves. Everything works correctly. :)
Have a good day!
